CHANGELOG.md

```diff
@@ -2,55 +2,70 @@
 
 ## Version 0.10.0 ⚓️ Anchor
 
-[**Release v0.10.0, (date goes here)**](Link to release goes here)
+<!-- [**Release 0.10.0, (<date-here>)**](https://github.com/OpenSTFoundation/mosaic-contracts/releases/tag/0.10.0) -->
+
+A gateway for a given EIP20 token is comprised of a `EIP20Gateway` contract on origin  and a corresponding `EIP20CoGateway` contract on auxiliary.
+On auxiliary there is an ERC20 utility token contract that mints and burns utility tokens.
+It atomically mirrors tokens staked and unstaked on the origin chain.
+
+Atomicity is achieved using a 2-phase message passing architecture between the chains.
+Messages are declared on the source chain and confirmed on the target chain.
+In order to confirm a message, a facilitator uses a Patricia Merkle proof once the source chain's state has been transferred to the target.
```

Comment:
Merkle Patricia proof instead of Patricia Merkle proof?

Reply:
True!